packages/apps/src/initSettings.ts

```diff
@@ -7,9 +7,18 @@ import store from 'store';
 import { createWsEndpoints } from '@polkadot/apps-config';
 import { extractIpfsDetails } from '@polkadot/react-hooks/useIpfs';
 import { settings } from '@polkadot/ui-settings';
+import { Endpoint, EndpointType } from '@polkadot/ui-settings/types';
 import { assert } from '@polkadot/util';
 
-function getApiUrl (): string {
+function networkOrUrl (apiType: Endpoint): void {
+  if (apiType.type === 'json-rpc' as EndpointType) {
```

Comment:
Some more `as EndpointType` - really need to see why we need to do this. A single place to coerce when we already know would be fine, but we have it everywhere when working with this type.